dags/analyzer_helper/discord/discord_transform_raw_data.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+from dags.analyzer_helper.discord.transform_raw_data_base import TransformRawDataBase
+
+
+class DiscordTransformRawData(TransformRawDataBase):
+    def transform(self, raw_data: list, platform_id: str, period: datetime) -> list[dict]:
+        transformed_data = []
+        for data in raw_data:
+            interactions = []
+            if data.get("replied"):
+                interactions.append({
+                    "name": "reply",
+                    "users_engaged_id": [data.get("replied")],
+                    "type": "receiver"
```

Comment:
If the field `replied_user` had a value the user will be emitter rather than receiver.

For the receiver case, we would assign the user id in `replied_user` to be receiver in another document.